.github/workflows/promote_release.yml

```diff
@@ -47,62 +39,71 @@ jobs:
       - name: Validate Gradle wrapper
         uses: gradle/wrapper-validation-action@v1
 
-      - name: Remove prerelease tag
-        run: |
-          echo "Removing prerelease tag from version"
-          echo "VERSION=$(echo ${{ steps.get-release.outputs.tag_name }} | sed -E 's/-pre.*//')" >> $GITHUB_ENV
-
-      - name: Build
+      - name: Build and test
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: clean build -x test -x checkstyleMain -x checkstyleTest
+          arguments: clean build -x assemble -x shadowJar
```

Comment:
Why change this?

Reply:
I think we should run the tests before promoting the release? Just incase something uncaught